Dev/Install/install.php5

```diff
@@ -152,7 +152,7 @@ if(isset($_REQUEST['db'])) {
 		$databaseConfig['type'] = $type;
 	}
 } else {
-	$type = $_REQUEST['db']['type'] = defined('SS_DATABASE_CLASS') ? SS_DATABASE_CLASS : 'MySQLDatabase';
+	$type = $_REQUEST['db']['type'] = defined('SS_DATABASE_CLASS') ? SS_DATABASE_CLASS : 'MySQLPDODatabase';
```

Comment:
Yes, hard to read :) And the installer is quite lengthy at 1700+ lines! I wonder if it should be broken into pieces, each dealing with only one aspect of the installation ?

It would be good to encourage pdo but fall back to the more common mysqli...how's about detecting pdo with:

`extension_loaded('PDO') && extension_loaded(;pdo_mysql')` 

Here are [more options](http://stackoverflow.com/questions/3131411/php-code-to-test-pdo-is-available).